tfs/src/main/java/hudson/plugins/tfs/TeamPushTrigger.java

```diff
@@ -119,13 +120,13 @@ public void run() {
             boolean shouldSchedule = bypassPolling;
             String changesDetected = "";
             if (!bypassPolling) {
-                if (runPolling()) {
+                if (runPolling() || StringUtils.isNotBlank(gitCodePushedEventArgs.commit)) {

```

Comment:
Sure, added comments below:
                // pipeline jobs might have runPolling() returned as false, while still should be scheduled.
                // we should shedule them as long as they are associated with a valid commit.